src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkMeta.tsx

```diff
@@ -0,0 +1,31 @@
+import { MetaTags } from "Components/MetaTags"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MyCollectionArtworkMeta_artwork } from "__generated__/MyCollectionArtworkMeta_artwork.graphql"
+
+interface MyCollectionArtworkMetaProps {
+  artwork: MyCollectionArtworkMeta_artwork
+}
+
+const MyCollectionArtworkMeta: React.FC<MyCollectionArtworkMetaProps> = ({
```

Comment:
I wonder if we could just use [ArtworkMeta](https://github.com/artsy/force/blob/main/src/Apps/Artwork/Components/ArtworkMeta.tsx#L15) here instead of creating a new component 🤔

Reply:
The only thing that made me create a new file is because the current ArtworkMeta page has link previews, which in my opinion is not a good thing at least for now, since we don't allow users to see other users' artworks

Reply:
Authorization should also work for the Metadata and it should be possible to see it for someone who hasn't created the artwork. 

@damassi any thoughts on this?